ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -170,6 +174,13 @@ protected BlockCreationResult createBlock(
       final ProtocolSpec newProtocolSpec =
           protocolSchedule.getByBlockHeader(processableBlockHeader);
 
+      final WithdrawalsProcessor withdrawalsProcessor = newProtocolSpec.getWithdrawalsProcessor();
+      maybeWithdrawals.ifPresent(
+          withdrawals ->
+              withdrawalsProcessor.processWithdrawals(withdrawals, disposableWorldState.updater()));
+
+      throwIfStopped();
```

Comment:
Interesting, I see this has moved to after withdrawals (compared to the prototype) - can you explain the significance please?

Reply:
There are multiple throwIfStopped calls. No significance in removing it,  think it makes sense to have it after as well, added that back in.